### PR TITLE
Feature/python3

### PR DESCRIPTION
**Pull request by CooperM125**
used 2to3 functionality in python for a majority of files.

### CODE REVIEW

File: nacc/uds3/fvp/builder.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ###############################################################################
 
 from nacc.uds3 import blanks
-import forms as fvp_forms
+from . import forms as fvp_forms
```
Comment (ufl-taeber) [inline nacc/uds3/fvp/builder.py]: PEP 8 recommends absolute imports. Where is `forms`? `nacc.uds3.fvp.forms`? If so
```suggestion
from nacc.uds3.fvp import forms as fvp_forms
```

**Comment by ufl-taeber** [comment]
I've only glanced over the changes, but they look pretty good! I'm glad **2to3** worked for the codebase.

Did you run the current, python2 nacculator a few times and compared the output with the python3 version? The output should match ***exactly***.

**Comment by CooperM125** [comment]
I have not compared the outputs of  python2 nacc vs python 3 nacc. That's a great idea!

**Comment by CooperM125** [comment]
Checked the outputs of nacc py2 and nacc py3 there was no difference.

File: nacc/uds3/__init__.py [hunk 2]
```diff
@@ -142,19 +142,19 @@ def form_name(self):
...
-            assert len(value) == end-start+1, \
-                "Length of field {} with value {} is not valid. {} != {}".format(field.name, value, len(value), end-start+1)
-            buf[start:start+len(value)] = value
+            assert len(value) == end - start + 1, \
+                "Length of field {} with value {} is not valid. {} != {}".format(field.name, value, len(value), end - start + 1)
+            buf[start:start + len(value)] = value.encode('ascii')
 
-        assert len(buf) == orig_buf_size, field.name + ": buffer changed size!"
-        return buf
+            assert len(buf) == orig_buf_size, field.name + ": buffer changed size!"
```
Comment (CooperM125) [inline nacc/uds3/__init__.py]: This should not be in the for loop

**Pull request merged**